### PR TITLE
Add `media sizes` command, to list registered media sizes

### PR DESCRIPTION
**Pull request by danielbachhuber**
Originally https://github.com/runcommand/media-sizes

### CODE REVIEW

File: src/Media_Command.php [hunk 1]
```diff
@@ -360,6 +360,82 @@ function import( $args, $assoc_args = array() ) {
 		}
 	}
 
+	/**
+	 * List media sizes registered with WordPress.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--fields=<fields>]
+	 * : Limit the output to specific fields. Defaults to all fields.
```
Comment (gitlost) [inline src/Media_Command.php]: Although it becomes clear as soon as you run it, it'd be nice to have a list of the fields here.

File: src/Media_Command.php [hunk 2]
```diff
@@ -360,6 +360,82 @@ function import( $args, $assoc_args = array() ) {
...
+				'crop'    => 'true',
+			),
+			array(
+				'name'    => 'thumbnail',
+				'width'   => intval( get_option( 'thumbnail_size_w' ) ),
+				'height'  => intval( get_option( 'thumbnail_size_h' ) ),
+				'crop'    => 'true',
+			),
+		);
+		foreach( $_wp_additional_image_sizes as $size => $size_args ) {
```
Comment (gitlost) [inline src/Media_Command.php]: I suppose for WPCS, there should be a space after the `foreach`.

File: src/Media_Command.php [hunk 3]
```diff
@@ -360,6 +360,82 @@ function import( $args, $assoc_args = array() ) {
...
+			$sizes[] = array(
+				'name'     => $size,
+				'width'    => $size_args['width'],
+				'height'   => $size_args['height'],
+				'crop'     => $size_args['crop'] ? 'true' : 'false',
+			);
+		}
+		usort( $sizes, function( $a, $b ){
```
Comment (gitlost) [inline src/Media_Command.php]: Also for WPCS, the opening `{` of the function should be preceded by a space and the closing `}` followed by one.

File: src/Media_Command.php (hunk 1 above)
Comment (danielbachhuber) [inline src/Media_Command.php]: bef97a31

File: src/Media_Command.php (hunk 2 above)
Comment (danielbachhuber) [inline src/Media_Command.php]: I'd rather not deal with minor coding standards issues until we have https://github.com/wp-cli/scaffold-package-command/issues/70 in place

Comment (gitlost) [inline src/Media_Command.php]: Okay, but if new stuff conforms, it'll be less churn.

**Pull request merged**